src/s2/s2region_term_indexer_test.cc

```diff
@@ -122,6 +122,9 @@ TEST(S2RegionTermIndexer, IndexRegionsQueryRegionsOptimizeTime) {
   options.set_max_level(16);
   options.set_max_cells(20);
   TestRandomCaps(options, QueryType::CAP);
+  TestRandomCaps(options, QueryType::POINT);
```

Comment:
This test case is called `IndexRegionsQueryRegions...`, so don't query points here.

Reply:
Ok, maybe needs to rename tests.

In general I think will be a lot of better to always test 3 cases:
query regions
query points
query points with points only optimization

Reply:
Fixed